app/kubernetes/converters/application.js

```diff
@@ -147,7 +148,8 @@ class KubernetesApplicationConverter {
     const persistedFolders = _.filter(res.Volumes, (volume) => volume.persistentVolumeClaim || volume.hostPath);
 
     res.PersistedFolders = _.map(persistedFolders, (volume) => {
-      const matchingVolumeMount = _.find(data.spec.template.spec.containers[0].volumeMounts, { name: volume.name });
+      const volumeMounts = _.uniq(_.flatMap(_.map(containers, 'volumeMounts')), 'name');
```

Comment:
If I'm right you can use `_.uniq(_.flatMap(containers, 'volumeMounts'), 'name')`